lib/packages_manager.py

```diff
@@ -84,5 +84,8 @@ def discover_packages():
     except OSError:
         LOG.error("No packages found in versions repository directory")
         raise
+        
+    if not package_list:
+    	raise exception.PackageDirectoryIsEmpty()
```

Comment:
I think it should also inform user what to do via `LOG.error` and then raise the exception.